cores/esp8266/core_esp8266_wiring_digital.cpp

```diff
@@ -125,11 +127,11 @@ typedef struct {
 	void* functionInfo;
 } ArgStructure;
 
-static interrupt_handler_t interrupt_handlers[16];
+static interrupt_handler_t interrupt_handlers[16] = { {0, 0, 0, 0}, };
```

Comment:
How does this init list even build? Shouldn't this be either {0} or {{0}, {0},... x16}?

Reply:
Why, the inner type (4 fields) is initialized by {0, 0, 0, 0}, for the remaining array the usual rules apply. Have you seen compiler warnings that I missed?

Reply:
Extra comma. I guess the compiler is forgiving here as in other cases.